diff --git a/chrome/browser/resources/certificate_viewer/certificate_viewer.html b/chrome/browser/resources/certificate_viewer/certificate_viewer.html
index 8967eee8d70..c625d9c2d06 100644
--- a/chrome/browser/resources/certificate_viewer/certificate_viewer.html
+++ b/chrome/browser/resources/certificate_viewer/certificate_viewer.html
@@ -40,4 +39,0 @@
-          <div>
-            <div class="attribute">$i18n{ou}</div>
-            <div id="issued-ou" class="value"></div>
-          </div>
@@ -57,4 +52,0 @@
-          <div>
-            <div class="attribute">$i18n{ou}</div>
-            <div id="issuer-ou" class="value"></div>
-          </div>
diff --git a/chrome/browser/ui/webui/certificate_viewer_ui.cc b/chrome/browser/ui/webui/certificate_viewer_ui.cc
index ed8fad1b17a..b88b5282027 100644
--- a/chrome/browser/ui/webui/certificate_viewer_ui.cc
+++ b/chrome/browser/ui/webui/certificate_viewer_ui.cc
@@ -38 +37,0 @@ void CreateAndAddWebUIDataSource(Profile* profile, const std::string& host) {
-      {"ou", IDS_CERT_INFO_ORGANIZATIONAL_UNIT_LABEL},
diff --git a/chrome/browser/ui/webui/certificate_viewer_webui.cc b/chrome/browser/ui/webui/certificate_viewer_webui.cc
index f899a8fecce..e1143beea1c 100644
--- a/chrome/browser/ui/webui/certificate_viewer_webui.cc
+++ b/chrome/browser/ui/webui/certificate_viewer_webui.cc
@@ -279,3 +278,0 @@ std::string CertificateViewerDialog::GetDialogArgs() const {
-    cert_info.SetByDottedPath(
-        "general.issued-ou",
-        HandleOptionalOrError(model.GetSubjectOrgUnitName()));
@@ -289,3 +285,0 @@ std::string CertificateViewerDialog::GetDialogArgs() const {
-    cert_info.SetByDottedPath(
-        "general.issuer-ou",
-        HandleOptionalOrError(model.GetIssuerOrgUnitName()));
diff --git a/components/certificate_matching/certificate_principal_pattern.cc b/components/certificate_matching/certificate_principal_pattern.cc
index 8b52046ff8e..35e884f0c44 100644
--- a/components/certificate_matching/certificate_principal_pattern.cc
+++ b/components/certificate_matching/certificate_principal_pattern.cc
@@ -87,0 +88 @@ CertificatePrincipalPattern CertificatePrincipalPattern::ParseFromOptionalDict(
+      // TOOD: remove OU here?
diff --git a/components/page_info/android/certificate_viewer_android.cc b/components/page_info/android/certificate_viewer_android.cc
index a368a5f3de6..b338598403d 100644
--- a/components/page_info/android/certificate_viewer_android.cc
+++ b/components/page_info/android/certificate_viewer_android.cc
@@ -38,6 +37,0 @@ JNI_CertificateViewer_GetCertInfoSerialNumberText(JNIEnv* env) {
-static ScopedJavaLocalRef<jstring>
-JNI_CertificateViewer_GetCertInfoOrganizationUnitText(JNIEnv* env) {
-  return ConvertUTF8ToJavaString(
-      env, l10n_util::GetStringUTF8(IDS_CERT_INFO_ORGANIZATIONAL_UNIT_LABEL));
-}
-
diff --git a/components/page_info/android/java/src/org/chromium/components/page_info/CertificateViewer.java b/components/page_info/android/java/src/org/chromium/components/page_info/CertificateViewer.java
index e74f285eeb0..0441617b553 100644
--- a/components/page_info/android/java/src/org/chromium/components/page_info/CertificateViewer.java
+++ b/components/page_info/android/java/src/org/chromium/components/page_info/CertificateViewer.java
@@ -162,2 +161,0 @@ public class CertificateViewer implements OnItemSelectedListener {
-        addItem(certificateView, CertificateViewerJni.get().getCertInfoOrganizationUnitText(),
-                sslCert.getIssuedTo().getUName());
@@ -172,2 +169,0 @@ public class CertificateViewer implements OnItemSelectedListener {
-        addItem(certificateView, CertificateViewerJni.get().getCertInfoOrganizationUnitText(),
-                sslCert.getIssuedBy().getUName());
@@ -291 +286,0 @@ public class CertificateViewer implements OnItemSelectedListener {
-        String getCertInfoOrganizationUnitText();
diff --git a/components/page_info_strings.grdp b/components/page_info_strings.grdp
index bce866203da..b0f7cfe92e5 100644
--- a/components/page_info_strings.grdp
+++ b/components/page_info_strings.grdp
@@ -639,3 +638,0 @@
-  <message name="IDS_CERT_INFO_ORGANIZATIONAL_UNIT_LABEL" desc="The label of the Organizational Unit field in the general page of the certificate info dialog.  (OU) is the name of this field in the standard">
-    Organizational Unit (OU)
-  </message>
diff --git a/weblayer/grit_strings_allowlist.txt b/weblayer/grit_strings_allowlist.txt
index 5c338e1836f..b8faf800cfc 100644
--- a/weblayer/grit_strings_allowlist.txt
+++ b/weblayer/grit_strings_allowlist.txt
@@ -79 +78,0 @@ IDS_CERT_INFO_ISSUER_GROUP
-IDS_CERT_INFO_ORGANIZATIONAL_UNIT_LABEL
