diff --git a/components/password_manager/core/browser/export/password_csv_writer.cc b/components/password_manager/core/browser/export/password_csv_writer.cc
index 54a864d848..a520f358ae 100644
--- a/components/password_manager/core/browser/export/password_csv_writer.cc
+++ b/components/password_manager/core/browser/export/password_csv_writer.cc
@@ -64,6 +64,14 @@ std::string PasswordCSVWriter::SerializePasswords(
     }
   }
 
+  std::sort(records.begin(), records.end(), [header](std::map<std::string, std::string>& a, std::map<std::string, std::string>& b){
+    for (const std::string& h : header) {
+      if (a[h] > b[h])
+        return false;
+    }
+    return true;
+  });
+
   std::string result;
   WriteCSV(header, records, &result);
   return result;
diff --git a/components/password_manager/core/browser/export/password_csv_writer_unittest.cc b/components/password_manager/core/browser/export/password_csv_writer_unittest.cc
index 365c6e76d8..b369c78a4c 100644
--- a/components/password_manager/core/browser/export/password_csv_writer_unittest.cc
+++ b/components/password_manager/core/browser/export/password_csv_writer_unittest.cc
@@ -148,17 +148,17 @@ TEST_P(PasswordCSVWriterTest, SerializePasswordsWritesNames) {
   std::string expected =
       base::FeatureList::IsEnabled(syncer::kPasswordNotesWithBackup)
           ? "name,url,username,password,note" + kLineEnding +
-                "example.com,https://example.com/,a,b," + kLineEnding +
                 "Netflix,android://Jzj5T2E45Hb33D-lk-"
                 "EHZVCrb7a064dEicTwrTYQYGXO99JqE2YERhbMP1qLogwJiy87OsB"
                 "zC09Gk094Z-U_hg==@com.netflix.mediaclient,a,b," +
-                kLineEnding
+                kLineEnding +
+                "example.com,https://example.com/,a,b," + kLineEnding
           : "name,url,username,password" + kLineEnding +
-                "example.com,https://example.com/,a,b" + kLineEnding +
                 "Netflix,android://Jzj5T2E45Hb33D-lk-"
                 "EHZVCrb7a064dEicTwrTYQYGXO99JqE2YERhbMP1qLogwJiy87OsB"
                 "zC09Gk094Z-U_hg==@com.netflix.mediaclient,a,b" +
-                kLineEnding;
+                kLineEnding +
+                "example.com,https://example.com/,a,b" + kLineEnding;
   EXPECT_EQ(expected, PasswordCSVWriter::SerializePasswords(credentials));
 }
 
